docs/index.md

```diff
@@ -1,154 +1,117 @@
+common developer activities in a convenient location, accessible from chat. These commands are code:
+consistent, shared, versioned. You might have gleaned that because your Software Delivery Machine is code, you're
+never limited to plugins, nor to anyone else's idea of the correct delivery mechanism; your SDM can integrate with your existing tools. [Keep the tools that are working for you][favorite-tools-blog], and integrate new ones
+as you wish. Tie these together with an SDM, plus bonuses like [automatic CHANGELOG management][changelog-pack].
+
+Your SDM gives you something else: the ability to manipulate code across all your projects with one command. 
+Atomist libraries help you write code transforms -- functions that operate on code -- and the API for Software
+turns those into branches, pull requests, or automatic commits on every deviating push. Bring code up to standards
+and then keep it there.
+
+Your SDM gives you something else: start new projects with the right code and setup, every time. Atomist generators
+start from a real, working project and use code transforms to construct a starting point in a new repository. Set up
+any other 
+
+Your SDM can respond to [more events], and to custom events that you send. You can query your correlated events
+using GraphQL. You can build chat commands, with updating messages and adding action buttons. 
+You can [add integrations][pack]
+and commands written by Atomist and the community, and contribute your own.
+
+[changelog-pack]: pack/changelog.md
+[pack]: pack/index.md
+[favorite-tools-blog]: https://the-composition.com/one-great-delivery-experience-your-favorite-tools-7f390f57d896 (One Great Delivery Experience, Your Favorite Tools)
+
+## Stories
+
+Check out some samples of what an Atomist SDM can do:
+
+*  [Deploy Spring Boot to Kubernetes][spring-boot-to-k8s-blog], and generate new projects, and manage versions of existing projects ([video][spring-one])
+*  [Upgrade test files to a new standard][autofix-blog-test]
+*  [Enforce custom code formatting][autofix-blog-stars]
+*  [Convert a migration script into a Slack command][schema-deploy-blog]
+
+[spring-one]: https://www.youtube.com/watch?v=VDCHnTPknsI (Rod Johnson speaks at Spring One 2018)
+[spring-boot-to-k8s-blog]: https://the-composition.com/deploy-your-spring-boot-application-to-kubernetes-in-3-mins-fdd37a212c6c (Christian Dupuis's Spring Boot to Kubernetes Blog)
+[autofix-blog-test]: https://the-composition.com/making-change-stick-with-code-transforms-and-autofixes-587d19e0ba1b (Rod blogs about upgrading test file names)
+[autofix-blog-stars]: https://the-composition.com/align-the-stars-programmatically-35dc5625f97d (Jess blogs about autofixing comment formatting)
+[schema-deploy-blog]: https://the-composition.com/automation-story-graphql-schema-deployment-7893eb55ed18 (Jess blogs about graphql schema deploy)
+
+## You might want to know
+
+*  [Existing integrations][integrations]
+*  [Architecture][]
```

Comment:
Will that work if the reference is all lower case?

Reply:
yes, it is not case-sensitive